src/Servers/Kestrel/Core/src/Internal/Http2/Http2FrameWriter.cs

```diff
@@ -541,7 +541,7 @@ public ValueTask<FlushResult> WritePingAsync(Http2PingFrameFlags flags, in ReadO
                     return default;
                 }
 
-                _outgoingFrame.PreparePing(Http2PingFrameFlags.ACK);
+                _outgoingFrame.PreparePing(flags);
```

Comment:
Caller always passes `Http2PingFrameFlags.ACK` in as `flags`; but if we are taking a `flags` parameter either should use it or drop the param.

Reply:
Worth a comment?

Reply:
It was odder before when the method's parameter was just ignored.